src/main/java/org/folio/event/handler/InventoryUpdateAsyncRecordHandler.java

```diff
@@ -74,6 +80,12 @@ private static InventoryUpdateHolder createInventoryUpdateHolder(ResourceEvent r
       .build();
   }
 
+  // Prepare a DB client for particular tenant
+  private Future<Void> prepareProcessInventoryUpdateEvent(InventoryUpdateHolder holder, String centralTenantId) {
+    holder.setCentralTenantId(centralTenantId);
+    return processInventoryUpdateEvent(holder, createDBClient(holder.getActiveTenantId()));
+  }
```

Comment:
interesting case, what if we have ecs, but consortium setting disabled. if kafka send member tenant, and based on current approach it will try to search member tenant because central tenant is null. In the end dbClient can fint from member because it is localted in central tenants. For Create Consumer we didn't proceess event

Reply:
> interesting case, what if we have ecs, but consortium setting disabled. if kafka send member tenant, and based on current approach it will try to search member tenant because central tenant is null. In the end dbClient can fint from member because it is localted in central tenants. For Create Consumer we didn't proceess event

Yeah I see, the alternative is to throw an exception or stop processing this event by silently returning

Reply:
Disabling this setting is not possible, at least for now, disabling is also not allowed from UI